lib/std/tempfiles.nim

```diff
@@ -82,26 +82,34 @@ template randomPathName(length: Natural): string =
     res[i] = state.sample(letters)
   res
 
+proc getTempDirImpl(dir: string): string =
```

Comment:
```suggestion
proc getTempDirImpl(dir: string): string {.inline.} =
```
Because it is called multiple times

Reply:
yeah I mean `genTempDir` too

Reply:
done, but please read https://github.com/nim-lang/RFCs/issues/198#issuecomment-596968252, which shows that `{.inline.}` impacts whether cross-module inlining can occur (in absence of -flto); since that proc is internal, it shouldn't really matter, but i suppose it's cleaner for intent + for robustness against refactorings.

Reply:
yeah I mean `genTempDir` too